### ISSUE TITLE
Portal from SVG wrapper to DIV

### ISSUE THREAD

**Issue by AnyCoder** [LinusBorg/portal-vue#287]
Hi!
I'm recently testing the possibilities of Portal Vue, found one thing that took me a bit of time to portal DIV element wrapped by SVG, altough the component's content is moved nicely in the DOM tree.

To picture it out briefly:

**Parent**
```
  <div style="position:absolute; top:50px; left:50px; z-index:9999">
    <portal-target name="destination" />
    <svg>
      <Child />
    </svg>
  </div>
```

**Child** as an independent SVG component, that additionally is supposed to render given content after click:
```
<template>
  <svg>
    <rect 
        width="100" height="100"
        @click="renderModal = !renderModal" 
    />
    <template v-if="renderModal">
        <portal to="destination">
          <div>Content</div>
        </portal>
    </template>
  </svg>
</template>
```

I noticed that the browser's useragent  adds to portaled child:
```
:not(svg) {
    transform-origin: 0px 0px;
}
```

instead of:
```
div {
    display: block;
}
```

which should show portaled element as a DOM element. I tried overidding CSS, scoping with slots or stripping the wrapper with `slim`, but not successfully. Same way with SVG side to strip it, like using `foreignObject`, but neither not working :( 

Do you maybe have any idea to workaround and make it able to mix with Portals nicely?

**Comment by LinusBorg** [comment]
Huh.

This is super weird. No idea where to start, honestly...

**Comment by LinusBorg** [comment]
closed in favor of #290
